metricbeat/module/kubernetes/state_persistentvolumeclaim/state_persistentvolumeclaim.go

```diff
@@ -29,6 +29,7 @@ var mapping = &p.MetricsMapping{
 	Metrics: map[string]p.MetricMap{
 
 		"kube_persistentvolumeclaim_access_mode": p.LabelMetric("access_mode", "access_mode"),
+		"kube_persistentvolumeclaim_created":     p.Metric("created.sec"),
```

Comment:
We collect the creation time in other metricsets of kubernetes module as well. See `state_service`, `state_job` and `state_storageclass`. In all those cases we use 
```
"kube_service_created":                      p.Metric("created", p.OpUnixTimestampValue()),
```
which transforms the seconds in a unix timestamp and declare the field as date. See https://github.com/elastic/beats/blob/9f93f932e9ae574a3e718ddb4d6a88aab63ffd8f/metricbeat/module/kubernetes/state_service/_meta/fields.yml#L31

I would suggest to do the same here and name the new filed `created` instead of `created.sec`